src/com/google/javascript/rhino/Messages.properties

```diff
@@ -235,6 +235,9 @@ msg.jsdoc.externs =\
 msg.jsdoc.nocompile =\
     extra @nocompile tag
 
+msg.jsdoc.nocollapse =\
```

Comment:
Add test in JsDocInfoParserTest.java that has two <code>@nocollapse</code> in a jsdoc comment.